### DISCUSSION TITLE
Should license be more permissive? Copyright assigned?

### DISCUSSION THREAD

**Discussion by scottveirs** [orcasound/orcasite#181] [General]
The current license is aGPL with no copyright assigned. Most recent Orcasound repositories have utilized the MIT license; a few have specified Orcasound as a copyright holder, e.g.:

* [OrcaAL license](https://github.com/orcasound/orcaal/blob/master/LICENSE)
* [Orca-action-workflow license](https://github.com/orcasound/orca-action-workflow/blob/main/LICENSE)

**Describe the solution you'd like**
A discussion of open source licensing in #dev-general (Orcasound Slack channel). Then, possibly, a shift in licensing here.

**Describe alternatives you've considered**

* Copyright held by contributors
* Copyright held by an over-arching foundation
* Pre-agreements about licensing and copyright within on-boarding materials?

**Comment by dthaler** [comment]
This was discussed live in the Oct. 16, 2024 orcasite meeting.  See the [notes](https://docs.google.com/document/d/1zXngvGO5kdm24gqSTNfHleFifU6ldtbA4amGSXe5jpg/edit?tab=t.0) from that meeting.
